apps/entity-service/src/entities/processors/entity-processor.ts

```diff
@@ -207,6 +222,19 @@ export abstract class ReportProcessor<
     await super.update(model, update);
   }
 
+  /**
+   * Creates a new report entity with the provided projectUuid.
+   * This method can be overridden in concrete report processors if custom creation logic is needed.
+   */
+  async create(attributes: CreateDto): Promise<ModelType> {
+    const model = await this.findOne(attributes.entityUuid);
+    if (model == null) {
+      throw new BadRequestException(`${this.resource} with UUID ${attributes.entityUuid} not found`);
+    }
+
+    return model;
+  }
```

Comment:
This is doing the same thing as the parent default implementation and should be removed. Furthermore, I think the default implementation should simply throw an error:
```
throw new BadRequestException("Creation not supported for this entity type");
```